### ISSUE TITLE
Examin Fallacy of Implied Continued action

### ISSUE THREAD

**Issue by pawn002** [pawn002/applied-ethics-in-design#9]
If I do something I must continue to do it even if doing so harms me, therefore I shouldn’t do it. 

Incorrect: doing something once is often good enough or better than nothing. 

e.g. charity

**Comment by coreygearhart** [comment]
an organizational "bias toward action" seems to have fallen out of favor recently, in favor of moderating action with reflection. 
- https://www.thinkingcollaborative.com/stj/bias-action/

I think this may have been an overcorrection though, I personally think leaning into action works best for me.
- https://www.benbowmanblog.com/home/2017/10/24/bias-toward-action
- https://medium.com/a-journey-to-design-thinking/design-mindsets-756bc69aed0d

@pawn002 is this the sort of thing you're getting at, or am I getting off topic?

**Comment by LonnyGomes** [comment]
@pawn002 Agreed, but I suppose the logic hinges on the definition of harm. The financial harm incurred by donating to charity is different to the harm caused by say jumping from a tree and breaking your arm. It seems that when many use such an argument they equate being uncomfortable to harm yet often that is a prerequisite to growth. 

I like @coreygearhart breakdown of bias to action. It made me thing would would be the opposite? Would it be to be risk averse? Perhaps it’s possible to become too reflective and become trapped in analysis paralysis.

**Comment by pawn002** [comment]
I find myself struggling to comment on both your insights; I think this is great!

My original intent in bringing up this issue was that I want to avoid people saying something is not worthwhile if you can’t keep doing it.

For example, If I give to charity I must continue to give to charity even if I can’t do it without harming myself. In this case, its still good to do it one time then never at all.

Extending this to Accessibility, some people have said that if we do this for a particular product then we have to do it all the time. I say yes, we should do it whenever we can (which is most of the time btw). However self harm can still negate doing it say for your stakeholders and such.

@coreygearhart Having clarified my intent a bit, do you mind extending your thoughts from above to these two examples?
